### PR TITLE
Li Tieg updates: A few new pieces of medical gear, clothing changed

### PR DESCRIPTION
**Pull request by Calyxman**
## About The Pull Request

The Li Tieg has been changed in terms of equipment. Medical belts and other clothing are replaced with their Trauma Team equivalents, while equipment and guns have remained relatively unchanged.
Images of changes:
image
image
image

Captain wearing the TT jumpsuit: 
image
CMO wearing the TT jumpsuit, hat and gloves:
image
TTT wearing the TT jumpsuit, hat and gloves: 
image
MI wearing the TT jumpsuit and hat: 
image
.. and also the Jr. Chemist jumpsuit:
image

Full list of changes:
Added 6 trauma team caps to the medical intern's locker.
Added 3 medical webbings to the medical intern's locker.
Gave the Captain locker and the CMO locker one trauma team cap and one jumpsuit.
Gave the CMO locker a medical webbing and red nitrile gloves.
Gave the equipment locker 3 red nitrile gloves, 3 medical webbings.
Gave the clothing locker 3 trauma team caps, and 6 trauma team jumpsuits.
Replaced the medkit crate with a nanomed; added a syringe box to the secure medical locker.
Added a wall mounted nanomed to the lobby.
Added a medical kiosk to the lobby. Now you really have that medical receptionist feel!
Trauma team technicians now get medHUD sunglasses, their trauma team uniforms and caps, as well as medical webbing and red nitrile gloves.
Added 3 junior chemist jumpsuits to the medical intern's locker.

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game

The Cybersun equipment really isn't seen much, it's only on one ship, and the Li Tieg could use some tweaks.
..Plus, it's called Trauma Team Technician yet there's no Trauma Team colors? This is a nice way to add some more style to the Li Tieg.

## Changelog

:cl:
tweak: Replaced the various nitrile gloves, belts and jumpsuits with their Trauma Team variants on the Li Tieguai.
add: Added a nanomed in place of the medical kit crate on the Li Tieg.
add: Added Trauma Team (cybersun subtype) clothes and red nitrile gloves to the Li Tieg, along with their hats.
add: Added a wall-mounted nanomed to the Li Tieg
add: Added some junior chemist clothes to the medical intern locker of the Li Tieg.
del: Removed the medical kit crate from the Li Tieg
tweak: Trauma Team Technicians on the Midway and the Li Tieguai come with Trauma Team gear.
/:cl:

### CODE REVIEW

**Review by rye-rice — CHANGES_REQUESTED** [review]
Take it upstream or modularize it.

**Comment by Vexylius** [comment]
Apologies, but I would prefer if you PRd it to [Shiptest(our upstream)](https://github.com/shiptest-ss13/Shiptest) instead. If you want it here only, please explain why

**Comment by Calyxman** [comment]
> Apologies, but I would prefer if you PRd it to [Shiptest(our upstream)](https://github.com/shiptest-ss13/Shiptest) instead. If you want it here only, please explain why

It'd be a lot of work just to copy this over it seems since I'm inexperienced, I asked in #ruinyard how they'd feel about me PRing it there and I got no reply, and.. this sounds weird but there's a certain level of trust with SE players (and how they'd use the gear 'responsibly') that I don't think I have for ST players. There's a big nanomed and this ship is already decently stocked. It does take away the medkit crate in exchange. And I still just don't feel comfortable to port it to a far bigger server, even then.

TLDR: I don't feel comfortable with porting it over, I also haven't gotten any reply about me porting it, and it took a while for me to implement the changes considering my inexperience.

**Comment by Calyxman** [comment]
Don't think I'm going to resolve merge conflicts for this PR, so.
